### ISSUE TITLE
Number() gives exponent formatting sometimes

### ISSUE THREAD

**Issue by plarsson** [bojanz/currency#25]
When using the RoundTo method with a precision of 8 and RoundHalfUp rounding mode on a zero amount, the Number method returns "0E-8" instead of "0.00000000". This inconsistency in formatting can cause confusion. Here's a snippet to reproduce the issue:

```
package main

import (
	"fmt"
	"github.com/bojanz/currency"
)

func main() {
	c, _ := currency.NewAmountFromInt64(0, "SEK")
	c = c.RoundTo(8, currency.RoundHalfUp)
	fmt.Printf("%s\n", c.Number())
}
```

The apd.Decimal String() defaults to use d.Text('G') which is intended?

> //	'G'	like 'E' for large exponents, like 'f' otherwise

One fix would be to change Number() to use apd.Decimal.Text('f') method but that will be a breaking change

It would be nice to be able to output Number() without exponents.

**Comment by bojanz** [comment]
This got fixed in cockraochdb/apd 3.2.0: https://github.com/cockroachdb/apd/pull/128

I updated our version yesterday: https://github.com/bojanz/currency/commit/a0d921405f5c34a10f2bc04c8f72e7199aa32ba2 but haven't tagged a release yet. I'll do so now.

**Comment by bojanz** [comment]
Released [v1.1.3](https://github.com/bojanz/currency/releases/tag/v1.1.3), let me know if you encounter any further issues.

**Comment by plarsson** [comment]
Works fine. Thanks for quick response!
